src/firefly/js/ui/DropDownContainer.jsx

```diff
@@ -42,7 +42,7 @@ export const dropDownMap = {
     MultiTableSearchCmd: {view: <MultiSearchPanel/>,  layout: {width: '100%'}},
     FileUploadDropDownCmd: {view: <FileUploadDropdown />, layout: {width: '100%'}},
     WorkspaceDropDownCmd: {view: <WorkspaceDropdown />},
-    DLGeneratedDropDownCmd: {view: <DLGeneratedDropDown/>, layout: {width: '100%'}},
+    DLGeneratedDropDownCmd: {view: <DLGeneratedDropDown key='DLGeneratedDropDownCmd' loadRegistry={false}/>, layout: {width: '100%'}},
```

Comment:
Why do you need a key here, especially when it's a constant?  If this is not a react's `key`, then maybe name it something else.

Reply:
key make it reset. If there is more than one the useState() will not reset.